### PR TITLE
Admin "gen magic code" method

### PR DESCRIPTION
**Pull request by markyfyi**
Users who owned their auth email stack were asking for a magic code helper to integrate Instant into their emails.

### CODE REVIEW

File: server/src/instant/model/instant_user.clj [hunk 1]
```diff
@@ -96,6 +96,10 @@
    (sql/execute-one! conn
                      ["DELETE FROM instant_users WHERE email = ?" email])))
 
+(defn get-or-create-by-email! [{:keys [email]}]
+  (or  (get-by-email {:email email})
```
Comment (stopachka) [inline server/src/instant/model/instant_user.clj]: nit: space

File: server/src/instant/admin/routes.clj [hunk 2]
```diff
@@ -263,6 +265,18 @@
   (let [{user-id :id app-id :app_id} (req->app-user! req)]
     (response/ok {:deleted (app-user-model/delete-by-id! {:id user-id :app-id app-id})})))
 
+(defn magic-code-post [req]
+  (let [email (ex/get-param! req [:body :email] email/coerce)
+        {user-id :id} (instant-user-model/get-or-create-by-email! {:email email})
```
Comment (stopachka) [inline server/src/instant/admin/routes.clj]: Afaik this is for the wrong model. This should be the `app-user-model` no?

**Review by stopachka — COMMENTED** [review]
Change to app-user-model and i.m.o G2G!

File: server/src/instant/admin/routes.clj (hunk 2 above)
Comment (markyfyi) [reply server/src/instant/admin/routes.clj]: :o

**Pull request merged**